reconcile/signalfx_endpoint_monitoring.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import sys
+from typing import Any, Optional
+
+from reconcile.closedbox_endpoint_monitoring_base import (
+    EndpointMonitoringProvider,
+    Endpoint,
+    parse_prober_url,
+    run_for_provider,
+)
+from reconcile.utils.openshift_resource import OpenshiftResource
+from reconcile.utils.semver_helper import make_semver
+
+QONTRACT_INTEGRATION = "signalfx-prometheus-endpoint-monitoring"
+QONTRACT_INTEGRATION_VERSION = make_semver(0, 1, 0)
+
+LOG = logging.getLogger(__name__)
+
+PROVIDER = "signalfx"
+
+
+def run(
+    dry_run: bool, thread_pool_size: int, internal: bool, use_jump_host: bool
+) -> None:
+    try:
+        run_for_provider(
+            provider=PROVIDER,
+            probe_builder=build_probe,
+            integration=QONTRACT_INTEGRATION,
+            integration_version=QONTRACT_INTEGRATION_VERSION,
+            dry_run=dry_run,
+            thread_pool_size=thread_pool_size,
+            internal=internal,
+            use_jump_host=use_jump_host,
+        )
+    except Exception as e:
+        LOG.error(e)
+        sys.exit(1)
+
+
+def build_probe(
+    provider: EndpointMonitoringProvider, endpoints: list[Endpoint]
+) -> Optional[OpenshiftResource]:
+    signalfx = provider.signalFx
+    if not signalfx:
+        return None
+    prober_url = parse_prober_url(signalfx.exporterUrl)
+    prober_url["path"] += "/" + signalfx.targetFilterLabel
+
+    relabeling = [
```

Comment:
Maybe a quick comment explaining this nice relabeling configuration?

Reply:
good idea. this needs some explanation! [86f77be](https://github.com/app-sre/qontract-reconcile/pull/2268/commits/86f77befba25dc49e6c6d8ce68df5197726f514d)